packages/api-client/src/socket/node.ts

```diff
@@ -14,12 +14,12 @@
 //
 
 import { type ClientSocket, type ClientSocketFactory } from '@hcengineering/client'
+import WebSocket from 'ws'
```

Comment:
Nope, no way, since package is intended to be also used from browser, and 'ws' is for nodejs websocket.